distribution/tools/upgrade-cli/build.gradle

```diff
@@ -14,6 +14,7 @@ archivesBaseName = 'opensearch-upgrade-cli'
 dependencies {
   compileOnly project(":server")
   compileOnly project(":libs:opensearch-cli")
+  implementation project(":distribution:tools:plugin-cli")
```

Comment:
Let's not create a dependency on `plugin-cli` - instead read from plugins.txt in `upgrade-cli`

Reply:
Removed the dependency on `plugin-cli`.